corehq/apps/users/management/commands/nphcda_find_mismatches.py

```diff
@@ -0,0 +1,151 @@
+import csv
+import re
+from collections import namedtuple
+from typing import Iterable, TypedDict
+
+from django.core.management.base import BaseCommand
+
+from corehq.apps.locations.models import SQLLocation
+from corehq.apps.users.models import CommCareUser
+
+
+Location = namedtuple('Location', 'id name code')
+country = Location('abc123', 'Nambia', 'nambia')
+location_cache = {
+    'nambia|foo|bar|baz': 'def456',
```

Comment:
Uses simple dictionary cache to reduce impact on the SQLLocation table. Some values can be prepopulated when location names in the file specified by the `users_csv` command line parameter do not match SQLLocation names.